### ISSUE TITLE
Crash on 11.1.0.34U New 3DS when injecting save with sploit program

### ISSUE THREAD

**Issue by MYCRAFTisbest** [yellows8/oot3dhax#83]
I have a New 3DS on version 11.1.0-34U and an Original 3DS on version 10.7.0-32U. I have been trying to use the OoThax sploit program on the o3ds to update the payloads. After properly setting everything up (changing the version, ect.) the game ether crashes and reboots the system, crashes to a black screen, or freezes. The exploit screen never actually appears. All the proper files are loaded on the system's microSD. 

Using the program to install the payload for 10.7 o3ds works perfectly for booting the o3ds to the homebrew launcher. I have tried using Ninjahax and Basicsploit to get to the homebrew launcher to install the payload and both have failed for installing for N3DS 11.1.

Is the sploit installer not up to date or is something else happening?

**Comment by MYCRAFTisbest** [comment]
I doubt this is helpful but the N3DS is the one from the Happy Home Designer bundle (so not an XL)
Technically I tried installing the exploit from a O3DS and a 2DS.

**Comment by yellows8** [comment]
On the Old3DS, did you select NEW 11-1-0-34-USA?

**Comment by MYCRAFTisbest** [comment]
Yes. I selected NEW 11-1-0-34-USA. I tried multiple times in case I was miss selecting something.

Out of curiosity, I tried using the NEW 11-1-0-34-USA file on the O3DS. Obviously it didn't work, but it made it to the Robin is ready screen, which is more than I can say for the N3DS which just freezes up the game or crashes.

**Comment by MYCRAFTisbest** [comment]
I thought a video might be helpful. [https://youtu.be/0O6at0Z8xKY](url) I apologize in advance for the hard to read text. The camera has a manual focus and I didn't want to mess with the current calibration I have it set to.

All the files on the N3DS microSD are up to date. I cleaned it off of everything I had on their besides the starterkit to try and rule out some conflicting file that might be present.

I appreciate you taking the time to help.

**Comment by yellows8** [comment]
Are you using the latest starter-kit?

**Comment by MYCRAFTisbest** [comment]
I'm getting the starterkit from http://smealum.github.io/3ds/

None of the files in there are newer than August, but I wouldn't know where else to get it.

**Comment by yellows8** [comment]
I can't repro this fwiw.

**Comment by MYCRAFTisbest** [comment]
Well thanks anyway. I'll tell you if anything changes.

Just to clarify, I should only need the starter-kit files and the payload is stored in the game's savefile.

**Comment by yellows8** [comment]
"Just to clarify, ..."Yes.

**Comment by deJEXUMN1AD2cGD7nwco8wtvTtsR** [comment]
@MYCRAFTisbest Do you have custom firmware? 

I can reproduce this with Clock+L2 enabled in Luma3DS (with OOT3D and Sticker). 

Turning off Clock+L2 in Luma settings allows things to execute normally. Hope this saves you some headache.

**Comment by MYCRAFTisbest** [comment]
@deJEXUMN1AD2cGD7nwco8wtvTtsR I appreciate it, but I don't have custom firmware.
I've since been able to get to the Homebrew Launcher with Cubic Ninja and also set up Stickerhax on my digital copy of the game, (which works fine).

I appreciate the effort though. Unless someone else comes forward having a similar issue, I wouldn't worry about this too much.

**Comment by Lawlipops** [comment]
I am also having the exact same issue (freezing or crashing when loading the save).

**Comment by MYCRAFTisbest** [comment]
@Lawlipops Are you using an older system to download the payload or are you using powersaves.

Also what system version are the systems?

**Comment by Lawlipops** [comment]
I'm using my friends older system to download the payload. I have an old 3DS XL on 11.1.0.34U and a New 3DS on 11.0.0.33U. I have tried it on both systems using their respective download files and I get the same error on both systems. I believe the New one is 11.0.0.33U but I'll check when I get home but the old 3DS XL should be on the latest firmware.

**Comment by MYCRAFTisbest** [comment]
Do you know what version your friends system is? Furthermore are you using an official game cart of OoT?

**Comment by Lawlipops** [comment]
I'll have to check when I get home, he's asleep and I'm at work haha. I am using the official game cartridge for OoT.

**Comment by Lawlipops** [comment]
His DS is the original 3ds on 10.7.0.32U firmware.

**Comment by MYCRAFTisbest** [comment]
The system I tried to use to install the payload was the same o3DS 10.7.0.32U Firmware. That seems to be the common factor as of now. 

Whether or not this is actually the root of the cause is beyond me. @yellows8 would probably know if the version of the system makes a difference or not.

**Comment by danne2** [comment]
I'm having the same issue on 11.0.0.34U on my N3DS. Tried both a powersaves install and a regular install via an old 3DS and it just boots me back to the home menu with "An error has occured"

**Comment by danne2** [comment]
After a good 20 attempts I just turned my 3d slider on and it worked first try. I'll see if I can repro it more

**Comment by yellows8** [comment]
"Edit: It seems the exploit really doesn't like ...." README

**Comment by danne2** [comment]
Edit: It seems that the exploit really doesn't like idiots who miss basic parts of the readme ;-)
